### ISSUE TITLE
Presubmit checks not showing up

### ISSUE THREAD

**Issue by drigz** [bazelbuild/continuous-integration#1095]
I'm not seeing the presubmit checks show on the glog repository:

- The webhook is configured to hit https://webhook.buildkite.com/deliver/...
- Presubmits are running (example: https://buildkite.com/bazel/google-logging/builds/1395)
- Write access is enabled for "bazel-io"
- However, I'm not seeing the buildkite check status on the PR: https://github.com/google/glog/pull/611

Has something changed here? IIRC I've had this problem once before, if this is common then we could add an FAQ for this.

**Comment by fweikert** [comment]
https://buildkite.com/bazel/google-logging/settings/repository displays the following error:

Buildkite has temporarily turned off commit status updates for this build pipeline.
We kept getting 404's when trying to make API requests.

This can happen if you've recently enabled third-party access restrictions.

In which case you'll need to connect your Github account to Buildkite via the Buildkite GitHub Application, and check that Buildkite has permissions to read this pipeline's repository.

Once you’ve done that, simply re-enable the Update commit statuses option below and we’ll start posting commit status updates again.

**Comment by drigz** [comment]
Thanks! Is it expected that I see "You don’t have the required access level to change this pipeline" when I try to load  the repo settings, or "You’re not allowed to change this organization" when I try to load "connect your Github accoutn to Buildkite"?

**Comment by fweikert** [comment]
I added you to the pipeline owners group. Can you please try again?

**Comment by drigz** [comment]
Since glog is in the google org, it seems like everything was set up already, and I just needed to reenable "Update commit statuses". Thanks for your help!

Aside: I noticed bazelci.py still points to the old glog-postsubmit.yml ([source](https://github.com/bazelbuild/continuous-integration/blob/2b4ee9f597d25a80800062224de55b8d4da056c3/buildkite/bazelci.py#L205)). This doesn't seem to be having an affect, since the Windows/MacOS builds (only configured in presubmit.yml in the glog repo) are indeed running, but maybe this is still worth cleaning up?
